lib/controller.dart

```diff
@@ -12,8 +12,29 @@ Map<String, model.Link> _navLinks = {
   'settings': model.Link('settings', 'Settings', view.renderSettingsTab)
 };
 
+const _errorMessages = {
+  'parse-config': 'Unable to parse "Config" to the required format',
+  'fetch-interactions': 'Unable to fetch interactions'
+};
```

Comment:
Sorry @elayabharath, I wasn't very clear previously, I meant something like:
```suggestion
const UNABLE_TO_PARSE_CONFIG_ERROR_MSG = 'Unable to parse "Config" to the required format';
const UNABLE_TO_FETCH_INTERACTIONS_ERROR_MSG = 'Unable to fetch interactions';
```

Reply:
Ah noted